client-local/src/main/java/com/cloudera/livy/client/local/LocalClient.java

```diff
@@ -381,6 +338,8 @@ public void run() {
       argv.add(serverAddress);
       argv.add("--remote-port");
       argv.add(serverPort);
+      argv.add("--config-file");
```

Comment:
It's a `RemoteDriver` option, so it means "the config file for the RemoteDriver" already. This will all become clearer when we're using the SparkLauncher API...